compiler/rustc_codegen_cranelift/scripts/cargo-clif.rs

```diff
@@ -13,8 +13,8 @@ fn main() {
     }
 
     let mut rustflags = vec!["-Cpanic=abort".to_owned(), "-Zpanic-abort-tests".to_owned()];
-    if let Some(name) = option_env!("BUILTIN_BACKEND") {
-        rustflags.push(format!("-Zcodegen-backend={name}"));
+    if let Some(_) = option_env!("BUILTIN_BACKEND") {
+        rustflags.push(concat!("-Zcodegen-backend=", env!("BUILTIN_BACKEND")).to_owned());
```

Comment:
This doesn't work. If `BUILTIN_BACKEND` is not set, this `env!()` will still report an error rather than take the code path below.